### PR TITLE
Add dockerfile for console plugin

### PR DESCRIPTION
**Pull request by savitaashture**

### CODE REVIEW

File: openshift/dockerfiles/console-plugin.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,32 @@
+ARG BUILDER=registry-proxy.engineering.redhat.com/rh-osbs/rhacm2-yarn-builder:rhel_8_nodejs_18_yarn_builder
```
Comment (vdemeester) [inline openshift/dockerfiles/console-plugin.Dockerfile:1]: I am not sure this will work (not behind the VPN).

**Review by vdemeester — COMMENTED** [review]
Could we re-use the `Dockerfile` on the root of this git repo ?

**Comment by savitaashture** [comment]
> Could we re-use the `Dockerfile` on the root of this git repo ?

Ya its almost same 

but we add some labels and all

and also the component which is coming from hack repo alwasy points to openshift folder so

File: openshift/dockerfiles/console-plugin.Dockerfile (hunk 1 above)
Comment (savitaashture) [reply openshift/dockerfiles/console-plugin.Dockerfile]: ya changed to registry.access.redhat.com/ubi8/nodejs-18:latest

**Comment by vdemeester** [comment]
The `Dockerfile` is also referenced by the `prow` build, and this is probably why `ci/prow/images` fails 😅

**Comment by savitaashture** [comment]
> The `Dockerfile` is also referenced by the `prow` build, and this is probably why `ci/prow/images` fails 😅

:thinking: 
how to fix this

**Comment by vdemeester** [comment]
@savitaashture symbolic link or duplicate ?

**Comment by savitaashture** [comment]
> @savitaashture symbolic link or duplicate ?

I mistakenly deleted original Dockerfile from root directory
added back now lets see

**Comment by vdemeester** [comment]
@savitaashture that works 😉 
/approve
/lgtm

**Pull request merged**